### PR TITLE
predix ui branding

### PR DESCRIPTION
**Pull request by irinaepshteyn**

### CODE REVIEW

File: uaa/src/main/resources/uaa.yml [hunk 1]
```diff
@@ -501,7 +581,11 @@ login:
 
 #  homeRedirect: http://example.com/
 
-#  branding:
+  branding:
+    companyName: Predix
+    footerLegalText:  "Copyright © 2020 General Electric Company. All rights reserved."
```
Comment (emarmol) [inline uaa/src/main/resources/uaa.yml:586]: Should this change to 2021 now or be dynamic?

File: uaa/src/test/resources/integration_test_properties.yml [hunk 2]
```diff
@@ -81,6 +82,11 @@ login:
       soTimeout: 10000
   authorize:
     url: http://localhost:8080/uaa/oauth/authorize
+  branding:
+    companyName: Predix
+    footerLegalText:  "Copyright © 2020 General Electric Company. All rights reserved."
```
Comment (emarmol) [inline uaa/src/test/resources/integration_test_properties.yml:87]: Same question about year here

**Comment by irinaepshteyn** [comment]
@emarmol, I made the requested changes

**Review by emarmol — APPROVED** [review]
lgtm (pending the functional review)

**Pull request merged**